packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -206,17 +199,24 @@ function NetworksPanel({
 
   const isNetworkSearchResult = Array.isArray(networksToShow)
 
-  const networkRowsWithChainInfoRows = useMemo(() => {
-    if (isNetworkSearchResult) {
-      return networksToShow
-    }
-    return [
-      ChainGroupName.core,
-      ...networksToShow.core,
-      ChainGroupName.orbit,
-      ...networksToShow.orbit
-    ]
-  }, [isNetworkSearchResult, networksToShow])
+  const networkRowsWithChainInfoRows: (ChainId | ChainGroupName)[] =
+    useMemo(() => {
+      if (isNetworkSearchResult) {
+        return networksToShow
+      }
+
+      const groupedNetworks = []
```

Comment:
We should always have at least one core chain but just in case we only show groups if they have at least one chain